docs/source/38_0.rst

```diff
@@ -0,0 +1,135 @@
+===============
+38.0: Love, Ken
+===============
+
+Sprint Theme
```

Comment:
Should we take the "sprint theme" this serious? I always took it as something internal, that we do just for fun. Should we be adding references to blockbuster movies such as "Terminator Genisys" (sprint #27) and "The Hateful Eight (#39) to the official documentation?

Maybe it could be a footnote instead of the first paragraph?